### PR TITLE
Add content images to all messages

### PR DESCRIPTION
**Pull request by cristian-uehara**
## Description
On this PR we added images for each type of message (anniversary, new hires, birthday) and included them into the message sent to Slack.
We used attachments and Slack's blocks to achieve it.

## Specific Changes
  - Added images
  - Added new templates to support randomly selecting an image. Right now there is one per message.
  - Updated message generator to generate the image URL.
  - Updated notify controller to generate an IActivity message with attachment using blocks.

### CODE REVIEW

File: birthday-bot/Controllers/NotifyController.cs [hunk 1]
```diff
@@ -190,17 +199,64 @@ private async Task SendBirthdayMessageToQueue()
...
+            {
+                Name = "blocks",
+                Content = blocks
+            };
+
+            var message = MessageFactory.Attachment(new List<Microsoft.Bot.Schema.Attachment> { attachment });
+            await turnContext.SendActivityAsync(message, cancellationToken);
         }
+
+        private List<Block> GetContentBlocks(string message, string image)
```
Comment (JuanAr) [inline birthday-bot/Controllers/NotifyController.cs:242]: I think this could be reviewed and moved to another class more coupled with Slack

**Review by erquirogasw — APPROVED** [review]

File: birthday-bot/Controllers/NotifyController.cs (hunk 1 above)
Comment (cristian-uehara) [reply birthday-bot/Controllers/NotifyController.cs]: Moved to the SlackInterop class. thanks

File: birthday-bot/Controllers/NotifyController.cs [hunk 2]
```diff
@@ -190,17 +198,44 @@ private async Task SendBirthdayMessageToQueue()
 
         private async Task BirthdayBotCallback(ITurnContext turnContext, CancellationToken cancellationToken)
         {
-            await turnContext.SendActivityAsync(happyBirthdayMessage);
+            var blocks = SlackInterop.GetContentBlocks(happyBirthdayMessage, happyBirthdayImage);
+
+            var attachment = new Microsoft.Bot.Schema.Attachment
```
Comment (JuanAr) [inline birthday-bot/Controllers/NotifyController.cs:203]: I see a refactor opportunity here

Comment (cristian-uehara) [reply birthday-bot/Controllers/NotifyController.cs]: Applied, thanks!

**Review by JuanAr — DISMISSED** [review]

**Pull request merged**